spec/android_device_spec.rb

```diff
@@ -1,4 +1,6 @@
 require 'device_api/android'
+require_relative 'helpers/stub_helper_spec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.